### PR TITLE
Updated README.md description to reflect new cosmetic name

### PR DESCRIPTION
**Pull request by Hedda**
Updated README.md description to reflect the new cosmetic name of "Enigma2"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
 [![Build Status](https://travis-ci.org/kodi-pvr/pvr.vuplus.svg?branch=master)](https://travis-ci.org/kodi-pvr/pvr.vuplus)
 [![Coverity Scan Build Status](https://scan.coverity.com/projects/5120/badge.svg)](https://scan.coverity.com/projects/5120)
 
-# VuPlus PVR
-VuPlus PVR client addon for [Kodi] (http://kodi.tv)
+# Enigma2 PVR
+PVR client addon for [Kodi] (http://kodi.tv) to control Enigma2-based TV-receivers/DVR/set-top boxes.
```
Comment (ksooo) [inline README.md]: https

Comment (Hedda) [reply README.md]: I have not changed the URL

Comment (phunkyfish) [reply README.md]: Yes, but the URL would be invalid. Can you change the http to https?

You could probably also do a single commit for these changes. Thanks

**Comment by phunkyfish** [comment]
There is also another mention of Vuplus to be removed: "Fetch picons from web interface: Fetch the picons straight from the Vuplus/Enigma 2 STB"

Also two more http instead of https if you want to fix those too ;)

[Kodi's PVR user support] (http://forum.kodi.tv/forumdisplay.php?fid=167)
[Kodi's PVR development support] (http://forum.kodi.tv/forumdisplay.php?fid=136)

If you want to make a single commit you can reset to master by running `git reset master`

Then commit the single change and force update to your fork: `git push <your-fork-url> -ff <your-branch-name>

Then it's nice and clean.

**Comment by Hedda** [comment]
Looks like ksooo might not accept the other PR so I'll leave this as a separate PR if you don't mind.

Please feel free to edit this PR as you wish before either merging or reject this PR. I'll leave it now.

**Comment by phunkyfish** [comment]
It might be easier if I just incorporate acceptable versions of your suggestions into my next PR.

Does that that sound ok with you?

**Comment by phunkyfish** [comment]
@Hedda, I've added you changes to my PR combined with other readme changes and added a credit to you in the changelog.

Hope that's cool with you.

**Comment by Hedda** [comment]
@phunkyfish That's cool with me! Thanks!

**Pull request closed without merging**